stable/nginx-ingress/templates/default-backend-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
```

Comment:
A user could do that using the current setup. Just need to set the `defaultBackend` values appropriately.